provider/azure/environ.go

```diff
@@ -1743,6 +1761,41 @@ func (env *azureEnviron) deleteResourceGroup(ctx context.ProviderCallContext, sd
 	return nil
 }
 
+func (env *azureEnviron) deleteResourcesInGroup(ctx context.ProviderCallContext, sdkCtx stdcontext.Context, resourceGroup string) error {
```

Comment:
should we delete for resources with model tag?

Reply:
This is the same as the regular behaviour for model destroy, just without the deletion of the resource group itself. All resources for a model are in the resource group.

Reply:
There's issues with deleting just resources with a given model tag - the Azure query APi is very limited in what you can use as an expression, eg you can't filter on tag name and value and resource group or even resource type. So we can't easily identify all the resources for a given model. e will document that the admin needs to create one resource group per model.